blackbox/docs/best_practice/full_restart_upgrade.txt

```diff
@@ -0,0 +1,175 @@
+.. highlight:: sh
+
+.. _full_restart_upgrade:
+
+====================
+Full Restart Upgrade
+====================
+
+The preferred way to upgrade a running CrateDB cluster is via the
+:ref:`rolling_upgrade` method. But this is not always an option (for example,
+when you are upgrading from one minor version to the next).
+
+When a :ref:`rolling_upgrade` is not possible, a full restart upgrade must be
+done if you want to update CrateDB.
+
+.. NOTE::
+
+   A full restart upgrade cannot be done without downtime for your database.
```

Comment:
it is obvious to us, sure. but I figure it might be a good idea to highlight this here so people who've never done one realise that the primary thing is they need to consider before attempting to do this. thoughts?

happy to remove if you don't think it's necessary